cosmos/operators/airflow_async.py

```diff
@@ -60,13 +55,11 @@ class DbtSourceAirflowAsyncOperator(DbtBaseAirflowAsyncOperator, DbtSourceLocalO
     pass
 
 
-class DbtRunAirflowAsyncOperator(BigQueryInsertJobOperator):  # type: ignore
+class DbtRunAirflowAsyncOperator(BigQueryInsertJobOperator, DbtRunLocalOperator):  # type: ignore
```

Comment:
It would be great if we can see the SQL statement in the operator Templated Field, like we can do with the Cosmos Local operators. If this is too much work, I'm happy for it to be done in a follow up PR/ticket.

Reply:
this din't work trivially for me. I have logged a follow-up ticket to work on it https://github.com/astronomer/astronomer-cosmos/issues/1490